core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -2403,6 +2403,7 @@ private void processQuery(
             long limit,
             @Transient SqlCompiler compiler
     ) throws PeerDisconnectedException, PeerIsSlowToReadException, QueryPausedException, BadProtocolException {
+        engine.getMetrics().pgWire().markStart();
```

Comment:
There is more systematic way of doing this.

There are two basic protocols in PG Wire, Simple and Extended. They have two separate entry points. We have test framework to make sure they are all covered.

That said, both PG wire entry points will end up in SqlCompiler. So will other end points, such as HTTP. It would be more prudent to record queries at that level